website/content/docs/agent/config/config-files.mdx

```diff
@@ -1586,15 +1586,95 @@ Valid time units are 'ns', 'us' (or 'µs'), 'ms', 's', 'm', 'h'."
+    [`raft_logstore.boltdb.no_freelist_sync`](#raft_logstore_boltdb_no_freelist_sync) field
+    instead.** Setting this to `true` will disable syncing the BoltDB freelist
+    to disk within the raft.db file. Not syncing the freelist to disk will
+    reduce disk IO required for write operations at the expense of potentially
+    increasing start up time due to needing to scan the db to discover where the
+    free space resides within the file.
+
+- `raft_logstore` ((#raft_logstore)) This is a nested object that allows
+  configuring options for Raft's LogStore component which is used to persist
+  logs and crucial Raft state on disk during writes. This was added in Consul
+  1.15.
+
+  - `backend` ((#raft_logstore_backend)) Specifies which storage
+    engine to use to persist logs. Valid options are `boltdb` or `wal`. Default
+    is `boltdb`. The `wal` option specifies an experimental backend that 
+    should be used with caution. Refer to 
+    [Experimental WAL LogStore backend](/consul/docs/agent/wal-logstore) 
+    for more information.
+
+  - `disable_log_cache` ((#raft_logstore_disable_log_cache)) This allows
+    disabling of the in-memory cache of recent logs. This exists mostly for
+    performance testing purposes. In theory the log cache prevents disk reads
+    for recent logs. In practice recent logs are still in OS page cache so tend
+    not to be slow to read using either backend. We recommend leaving it enabled
+    for now as we've not measured a significant improvement in any metric by
+    disabling.
+
+  - `verification` ((#raft_logstore_verification)) This is a nested object that
+    allows configuring online verification of the LogStore. Verification
+    provides additional assurances that LogStore backends are correctly storing
+    data. It imposes very low overhead on servers and is safe to run in
+    production, however it's mostly useful when evaluating a new backend
+    implementation.
+
+    Verification must be enabled on the leader to have any effect and can be
+    used with any backend. When enabled, the leader will periodically write a
+    special "checkpoint" log message including checksums of all log entries
+    written to Raft since the last checkpoint. Followers that have verification
+    enabled will run a background task for each checkpoint that reads all logs
+    directly from the LogStore and recomputes the checksum. A report is output
+    as an INFO level log for each checkpoint.
+
+    Checksum failure should never happen and indicate unrecoverable corruption
+    on that server. The only correct response is to stop the server, remove its
+    data directory, and restart so it can be caught back up with a correct
+    server again. Please report verification failures including details about
+    your hardware and workload via GitHub issues. Refer to 
+    [Experimental WAL LogStore backend](/consul/docs/agent/wal-logstore) 
+    for more information.
```

Comment:
I missed that. IMO would read more clearly as
```suggestion
    Checksum failure indicatea unrecoverable corruption
    on that server. If this occurs, stop the server, remove the data directory, and restart so it can re-sync its state with the leader again. Report each verification failure including details about
    your hardware and workload in a GitHub issue. Refer to 
    [Experimental WAL LogStore backend](/consul/docs/agent/wal-logstore) 
    for more information.
```

Does that read better to you @boruszak?

Reply:
Oops saw this was merged. Nevermind!